PaintingTrivia.py

```diff
@@ -125,35 +144,66 @@ def guess_the_painter():
             return x
 
         try:
-            chosen_painter = int(request.form['chosen_painter'])
+            quiz = pickle.loads(session['quiz'])
```

Comment:
afaik, following is the _pythonic_ way to get the values out of the dictionary. please change
`    try:
        quiz = pickle.loads(session['quiz'])
    except KeyError:
        quiz = PainterQuiz()
        session['quiz'] = pickle.dumps(quiz)
`